validator/usage.go

```diff
@@ -79,6 +79,8 @@ var appHelpFlagGroups = []flagGroup{
 			flags.KeyManager,
 			flags.KeyManagerOpts,
 			flags.KeystorePathFlag,
+			flags.MergeSourceDirectories,
+			flags.MergeTargetDirectory,
```

Comment:
I am not sure if these flags should be added here or not because I don't understand the purpose of this file.

Reply:
These are fine here, thanks @rkapka